docs/source/conf.py

```diff
@@ -974,6 +974,7 @@
     "is_symbolic",
     "parallel_and",
     "parallel_or",
+    "statically_known_true",
```

Comment:
No new entry here please, either properly document this so that it shows up on the website or make it private.

Reply:
This function is intended to be public, but the entire `torch.fx.experimental` module isn't documented so I thought it would be weird to add documentation just for this one function.

Reply:
We should document that module properly.